trade/trade_currency_for_currency.py

```diff
@@ -9,49 +9,69 @@
 from trade.trade_waits import wait_for_window
 from utils.chat_utils import trade_user
 from utils.resetCursor import reset_cursor
+from utils.prefix_print import printtime
+import pyperclip
+import re
+import time
 
-
-# 1. Trade user
-# 2. Check if waiting popup exists
-# 3. If popup disappears check if trade window exists
-# 4. If trade window exists proceed
-# 5. If user cancells kick and shit
-# 6. If trade window did not appear sleep for 5s and retry
-# 7. If still nothing happens kick the user
+cell_size=54
 
 def trade_currency_for_currency(trade_order, n=0):
     if n == 2:
         return
-
     sell_currency_config = CURRENCY_TAB[trade_order.sell_currency]
     sell_cells = math.ceil(trade_order.sell_amount / sell_currency_config['stackSize'])
-
     trade_user(trade_order.buyer)
     trade_started = wait_for_window()
-
     if trade_started:
         empty_equipment(sell_cells)
         reset_cursor()
-
-        try:
-            total_amount = 0
-            cell_index = 0
-            while total_amount < trade_order.buy_amount:
-                move_cursor_to_cell(cell_index)
-                cell_currency, cell_amount = get_cell_info_currency()
-
-                if cell_currency != trade_order.buy_currency:
-                    raise TradeException('Wrong currency')
-
-                total_amount += cell_amount
-                cell_index += 1
-        except Exception as e:
-            print(e)
-            pyautogui.press('ESC')
+        
+        check_times=0
+        if check_times<3:
+            try:
+                total_amount = 0
+                while total_amount < trade_order.buy_amount:
+                    pyautogui.PAUSE=0.001
```

Comment:
I believe you could just set this value at the start of the script (in `main.py`), please check if that would be enough without breaking added changes. If this change would work, please apply remove all other pause calls added.